x-pack/test/functional/apps/ml/anomaly_detection/anomaly_explorer.ts

```diff
@@ -64,7 +64,7 @@ export default function ({ getService }: FtrProviderContext) {
   const elasticChart = getService('elasticChart');
 
   describe('anomaly explorer', function () {
-    this.tags(['mlqa']);
+    this.tags(['mlqa', 'dima']);
```

Comment:
Leftover `'dima'` 😅

Reply:
fixed in https://github.com/elastic/kibana/pull/126274/commits/a6eefbefd5c57bead047bae6627b2af3c86c6a7f